persistence/in-memory/index.md

```diff
@@ -7,7 +7,20 @@ redirects:
 - nservicebus/persistence/in-memory
 ---
 
-Some scenarios require an in-memory persistence such as the development environment or a lightweight client not interested in durability across restarts:
+Some scenarios require an in-memory persistence such as the development environment or a lightweight client not interested in durability across restarts.
+
+## Persistence at a glance
+
+|Feature                    |   |
+|:---                       |---
+|Storage Types              |Sagas, Outbox, Subscriptions, Timeouts
+|Unsupported Storage Types  |None
+|Transactions               |None
```

Comment:
AFAIK saga persistence is transactional

Reply:
That is in the sentence below, other than that there are no transactions, afaik

Reply:
I don't care if the code technically has an Enlist thing in it, I am not saying an in-memory persistence has transactions. Fails the D of ACID hardcore.